cfpb/cfgov-refresh
# Add Related Metadata Molecule to the sidefoot for pages

- Changed browse-filterable/index.html and molecules/related-metadata.html according to new backend
- Topics added without markup

### Frontend work needed for the topic list

### Test
- Run `./cfgov/manage.py migrate`
- Go to add a new **DocumentDetail** page
- Navigate to the Sidebar (Footer) tab
- Add `Related Metadata` block content
 - To render related pages by tags select `Topics` 
![screen shot 2016-02-12 at 1 27 47 pm](https://cloud.githubusercontent.com/assets/254877/13016400/8678459e-d18c-11e5-9d07-754f1a792557.png)

 - make sure your related pages have the same tags as the current page for them to be displayed 
![screen shot 2016-02-12 at 1 28 53 pm](https://cloud.githubusercontent.com/assets/254877/13016412/94770ad6-d18c-11e5-9e74-9e9d9d37976f.png)

- Preview/Publish then view the page
![screen shot 2016-02-12 at 1 29 13 pm](https://cloud.githubusercontent.com/assets/254877/13016418/a21803fc-d18c-11e5-8419-3e4b30fcf706.png)


@sebworks 
@rosskarchner 
@richaagarwal
